SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -543,6 +543,48 @@ void FSpatialGDKEditorToolbarModule::SchemaGenerateFullButtonClicked()
 	GenerateSchema(true);
 }		
 
+void FSpatialGDKEditorToolbarModule::OnShowSingleFailureNotification(const FString& NotificationText)
+{
+	AsyncTask(ENamedThreads::GameThread, [NotificationText]
+		{
```

Comment:
Nit: move the lambda scope 1 tab back.
```suggestion
	{
```